opendocdb-cts/internal/mongosh/testsuite.go

```diff
@@ -39,3 +43,45 @@ func ConvertResponse(res *wirebson.Document) (string, error) {
 
 	return "response = " + s + "\n", nil
 }
+
+// ConvertTestSuites converts request and response from each testsuite
+// to a mongosh's JavaScript format, and writes them to files in specified
+// directory.
+func ConvertTestSuites(testSuites data.TestSuites, outDir string) error {
+	for tsName, ts := range testSuites {
+		reqDir := filepath.Join(outDir, "requests", tsName)
+		resDir := filepath.Join(outDir, "responses", tsName)
```

Comment:
I thought we are trying to avoid having `requests` and `responses`. By introducing changes, doesn't it create merge conflict on #35?

Reply:
The #35 merges changes from this PR, as it solves separate issue, I did the change in separate PR.

Reply:
It just looks like unnecessary refactoring of creating this function in this PR...